src/pretix/control/templates/pretixcontrol/event/settings.html

```diff
@@ -113,10 +113,15 @@ <h4>{% trans "Attendee data (once per personalized ticket)" %}</h4>
                 </div>
                 {% bootstrap_field sform.attendee_data_explanation_text layout="control" %}
 
-                <h4>{% trans "Other settings" %}</h4>
+                <h4>{% trans "Form settings" %}</h4>
                 {% bootstrap_field sform.name_scheme layout="control" %}
                 {% bootstrap_field sform.name_scheme_titles layout="control" %}
                 {% bootstrap_field sform.checkout_show_copy_answers_button layout="control" %}
+
+                <h4>{% trans "Changes to existing orders" %}</h4>
+                {% bootstrap_field sform.allow_modifications layout="control" %}
+                {% bootstrap_field sform.last_order_modification_date layout="control" %}
+                {% bootstrap_field sform.allow_modifications_after_checkin layout="control" %}
```

Comment:
die braucht man nur wenn allow_modifications nicht auf no ist, oder? 
```suggestion
                <div data-display-dependency='#id_settings-allow_modifications_0' data-inverse>
                    {% bootstrap_field sform.last_order_modification_date layout="control" %}
                    {% bootstrap_field sform.allow_modifications_after_checkin layout="control" %}
                </div>
```